### PR TITLE
docs - nginx client body size limitation

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

File: docs/deploy/k8s.md [hunk 1]
```diff
@@ -113,6 +113,9 @@ You should have a load balancer direct requests to the lakeFS server.
 Options to do so include a Kubernetes Service of type `LoadBalancer`, or a Kubernetes Ingress.
 By default, lakeFS operates on port 8000, and exposes a `/_health` endpoint which you can use for health checks.
 
+Nginx Ingress Controller has a default client's body size of 1 MB.
```
Comment (arielshaqed) [inline docs/deploy/k8s.md:116]:
```suggestion
The NGINX Ingress Controller by default limits the client body size to 1 MiB.
```

But I would consider calling it out more explicitly, e.g.
```suggestion
The NGINX Ingress Controller by default limits the client body size to 1 MiB.  This is not useful for working with large objects, and should be increased at least to 5 MiB to allow multipart uploads, and to much more for safe usage.
```

**Review by arielshaqed — APPROVED** [review]
Important stuff!

**Review by ozkatz — APPROVED** [review]
Approving because it's much better than before! Thanks :)

Some things I believe could improve it even further:

1. Explain which API calls benefit from this (i.e. put/get object, s3 gtw usage, etc) and why
2. give a concrete recommendation for a value for this setting (i.e. if Spark/Hadoop/S3A/?? has a setting to control the max size of data files, perhaps use that * 1.2, or any other formula we feel safe with)

**Comment by itaiad200** [comment]
Thanks for the review, pulling once checks pass

**Pull request merged**